warc_metadata_sidecar.py

```diff
@@ -27,6 +27,8 @@
 
 TEXT_FORMAT_MIMES = re.compile(r'(text|html|xml)')  # this may change
 
+ARC = re.compile(r'.*(\.arc)(\.gz)?')
```

Comment:
Two small suggestions:

1. We can remove the parentheses around `\.arc` since we're not really trying to capture anything here, just wanna know if we get a match or not.
2. I think we need a '$' after the '?' in order for things like "something.arctic" to not match.

So essentially I think it should look like this:
`ARC = re.compile(r'.*\.arc(\.gz)?$')`

Reply:
I guess we also could be using re.match instead of re.search, as we're only interested in full matches. If you go with that, I don't think we would need the "$"

Reply:
urgh, sorry, this is my bad, we _do_ need the "$" after the "?", even with match. But still, I think the move to using match was the right one.